### DISCUSSION TITLE
Install errors and update packaged server

### DISCUSSION THREAD

**Discussion by nielsnw** [TriliumNext/Trilium#2079] [Q&A] [answered]
Hi there,

I hope some noob questions are welcome here, still learning linux but I seem to get along better these days.

Today I tried to install the Trilium server on my Ubuntu 21.04 and this failed due serveral errors in the npm proces, I could not remediate the errors and so I installed the packeged version that seems to be working just fine.

I am wondering, is the packaged installation updated with apt-get? Or do I need the manual installation for that? In general: how do I deal with updating Trilium?

Possible I am missing something fundamental here, looking forward!

Kind regards, Niels

**Comment by zadam** [comment, marked as answer]
Trilium will not get updated automatically, you need to do it manually. So in case of packaged server version, you stop the existing trilium process, download and unzip the new version, run the new version ...

**Reply by nielsnw** [reply]
Cool thanks, one last thing if you do not mind.

Can you tell me a reason why the "nohup node src/www &" is not working? nohup: ignoring input and appending output to 'nohup.out'. When I run ./trilium.sh everything works, running the nohup command does not.

It seems I cannot log out of SSH with Trilium runnuing.

*** 

Sorry, seems I did not understand the nohup command. I got that working now.

***

Is there a cache active when I have logged onto Trilium with one workstation, that prevents me to access Trilium with another? When I try to access Trilium with a second computer the page is not timed-out.

Looking forward!

**Reply by nielsnw** [reply]
Got it, great app.
